stablehlo/dialect/TypeInference.cpp

```diff
@@ -3614,6 +3614,43 @@ LogicalResult verifyDotOp(std::optional<Location> location,
   return success();
 }
 
+LogicalResult verifyDotGeneralOpQuantizationConstraints(
+    std::optional<Location> location, Type lhsType, Type rhsType,
+    Type resultType, ArrayRef<int64_t> rhsContractingDimensions) {
+  Type lhsElementType = getElementTypeOrSelf(lhsType);
+  Type rhsElementType = getElementTypeOrSelf(rhsType);
+  Type resultElementType = getElementTypeOrSelf(resultType);
+
+  // dot_general_c15
+  if (auto rhsPerTensorQuantType =
+          dyn_cast<quant::UniformQuantizedType>(rhsElementType)) {
+    if (rhsPerTensorQuantType.getZeroPoint() != 0) {
+      return emitOptionalError(location,
+                               "Zero point of rhs of dot_general should be 0");
```

Comment:
omit op name from the error log here, and other places.

Reply:
Done.